CITATION.md

```diff
@@ -51,29 +55,9 @@ JOSS: https://joss.theoj.org/papers/10.21105/joss.00588
 }
 ```
 
-Zenodo:
+## Zenodo
 
-The latest version is always here:
 https://doi.org/10.5281/zenodo.1202434
 
-```
-@software{the_pynucastro_development_2022_7239007,
-  author       = {the pynucastro development and
-                  Boyd, Brendan and
-                  Cao, Lyra and
-                  Chen, Zhi and
-                  Eiden, Kiran and
-                  Johnson, Eric and
-                  Li, Xinlong and
-                  Smith Clark, Alexander and
-                  Willcox, Donald and
-                  Zingale, Michael},
-  title        = {pynucastro/pynucastro: pynucastro 2.0.2},
-  month        = oct,
-  year         = 2022,
-  publisher    = {Zenodo},
-  version      = {2.0.2},
-  doi          = {10.5281/zenodo.7239007},
-  url          = {https://doi.org/10.5281/zenodo.7239007}
-}
-```
+The bibtex for the latest version is always here:
+https://zenodo.org/records/13899727/export/bibtex
```

Comment:
Since `https://zenodo.org/records/1202434/export/bibtex` doesn't work, maybe something like this is better?
```suggestion
The bibtex for the latest version is included below (updated automatically):
```

Reply:
oh wait...

that's in `CHANGES.md`, so if someone views that independent of the docs, then it won't appear

Reply:
yeah, I'll do that